src/runtime_config/default_whitelists.h

```diff
@@ -32,7 +32,6 @@ namespace ccf
       Tables::MEMBER_CERTS,
```

Comment:
Members should have access to the nodes table via lua, same as they do to the members table, etc. If they want to change the way that the nodes are handled they should be able to propose and vote for that.

Reply:
The join protocol is implemented in C++ though, and we do want to enforce a particular set of conditions and transitions for nodes to preserve security guarantees. What's the use case for opening this up completely?

Reply:
Raised https://github.com/microsoft/CCF/issues/806